rabbit_context.py

```diff
@@ -9,21 +9,25 @@ class RabbitContext:
 
     def __init__(self, **kwargs):
         self._host = kwargs.get('host') or os.getenv('RABBITMQ_SERVICE_HOST', 'localhost')
-        self._port = kwargs.get('port') or os.getenv('RABBITMQ_SERVICE_PORT', '5672')
+        self._port = kwargs.get('port') or os.getenv('RABBITMQ_SERVICE_PORT', '6672')
```

Comment:
I don't think we should use RABBITMQ_SERVICE_PORT when we use RABBITMQ_PORT everywhere else

Reply:
The same variable names are used in the OPS tool at least